js/functions.js

```diff
@@ -503,17 +545,17 @@ function buttonVisibilityHandler(mRecallBtn, mClearBtn) {
   }
 }
 
-//== change in btn UI functions ==
+//== change in btn UI functions ==//
 
 //fn that changes color
-function changeButtonColor(e) {
+export function changeButtonColor(e) {
   e.target.classList.contains("btn-blue")
     ? e.target.classList.remove("btn-blue")
     : e.target.classList.add("btn-blue");
```

Comment:
Can we do this with `classList.toggle('className')` ?

Reply:
added the toggle instead of remove and add.